scheduling/storage_provider.go

```diff
@@ -2,12 +2,14 @@ package scheduling
 
 import (
 	"fmt"
+	"sort"
 	"strings"
 )
 
 type StorageProvider interface {
```

Comment:
does it make sense to store StorageProvider in scheduling? Won't it eventually be the abstraction used in coordinator as well?